boards/xmc-2go/Makefile.features

```diff
@@ -0,0 +1,10 @@
+FEATURES_MCU_GROUP = cortex_m0
```

Comment:
could you please use the same scheme as for e.g. the `boards/stm32f4discovery/Makefie.features` for this file? Would be nice to have them all in the same style...